src/common/services/user_skill_map.js

```diff
@@ -0,0 +1,37 @@
+import { FetchUserSkillMap } from "common/services/request/query/user_skill_map";
+
+const FetchUserSkillMapByUserId = (id) => {
+  const payload = { ...FetchUserSkillMap };
+  payload.where = {
+    clause: {
+      conditions: [
+        {
+          field: "user_id",
+          operator: "eq",
+          value: id,
+        },
+      ],
+    },
+  };
+  return payload;
+};
+
+const InsertUserSkillMap = (user_id, skill_id, expert_level) => {
+  return {
+    display: "insert User Skill Map",
+    name: "insert_meta_user_skill_map_one",
+    function: "insert_meta_user_skill_map_one",
+    write: true,
+    object: {
+      user_id: "03724f8f-e5c7-423d-a98c-a7fdee9e959f",
```

Comment:
Hoping this is only for the test purpose and the hard coding will be removed later?

Reply:
yes just for test purpose, will be updated ACC to the param we will receive in the url